File: listas de exercicio/listas 3/lista 3 sala C/questao 11.c
#include <stdio.h>

int main(){
int A, B, C, D, E;
    printf("Digite um numero inteiro para a:");
    scanf("%d", &A);
    printf("Digite um numero inteiro para b:");
    scanf("%d", &B);
    printf("Digite um numero inteiro para c:");
    scanf("%d", &C);
    printf("Digite um numero inteiro para d:");
    scanf("%d", &D);
    printf("Digite um numero inteiro para e:");
    scanf("%d", &E);

    
if (A>B && A>C && A>D && A>E){
    printf("o maior numero é %d",A);
    if (B<C && B<D && B<E){
        printf("o menor numero é %d",B);
    }
    if (C<B && C<D && C<E){
        printf("o menor numero é %d",C);
    }
    if (D<C && D<B && D<E){
        printf("o menor numero é %d",D);
    }
    if (E<C && E<D && E<B){
        printf("o menor numero é %d",E);
    }
}
if (B>A && B>C && B>D && B>E){
    printf("o maior numero é %d",B);
    if (A<C && A<D && A<E){
        printf("o menor numero é %d",A);
    }
    if (C<B && C<D && C<E){
        printf("o menor numero é %d",C);
    }
    if (D<C && D<B && D<E){
        printf("o menor numero é %d",D);
    }
    if (E<C && E<D && E<B){
        printf("o menor numero é %d",E);
    }
}
if (C>A && C>B && C>D && C>E){
    printf("o maior numero é %d",C);
    if (A<B && A<D && A<E){
        printf("o menor numero é %d",A);
    }
    if (B<A && B<D && B<E){
        printf("o menor numero é %d",B);
    }
    if (D<C && D<B && D<E){
        printf("o menor numero é %d",D);
    }
    if (E<C && E<D && E<B){
        printf("o menor numero é %d",E);
    }
}
if (D>A && D>C && D>B && D>E){
    printf("o maior numero é %d",D);
    if (A<C && A<D && A<E){
        printf("o menor numero é %d",A);
    }
    if (C<B && C<D && C<E){
        printf("o menor numero é %d",C);
    }
    if (B<C && B<A && B<E){
        printf("o menor numero é %d",B);
    }
    if (E<C && E<D && E<B){
        printf("o menor numero é %d",E);
    }
}
if (E>A && E>C && E>D && E>B){
    printf("o maior numero é %d",E);
    if (A<C && A<D && A<E){
        printf("o menor numero é %d",A);
    }
    if (C<B && C<D && C<E){
        printf("o menor numero é %d",C);
    }
    if (D<C && D<B && D<E){
        printf("o menor numero é %d",D);
    }
    if (B<C && B<D && B<E){
        printf("o menor numero é %d",B);
    }
}
}

File: listas de exercicio/listas 3/lista 3 sala python/questao 11.py
A = (int(input("Digite o primeiro valor:")))
B = (int(input("Digite o segundo valor:")))
C = (int(input("Digite o terceiro valor:")))
D = (int(input("Digite o quarto valor:")))
E = (int(input("Digite o quinto valor:")))

if A>B and A>C and A>D and A>E:
    print(f"o maior numero é {A}")
    if B<C and B<D and B<E:
        print(f"o menor numero é {B}")
    if C<B and C<D and C<E:
        print(f"o menor numero é {C}")
    if D<C and D<B and D<E:
        print(f"o menor numero é {D}")
    if E<C and E<D and E<E:
        print(f"o menor numero é {E}")
if B>A and B>C and B>D and B>E:
    print(f"o maior numero é {B}")
    if A<C and A<D and A<E:
        print(f"o menor numero é {A}")
    if C<B and C<D and C<E:
        print(f"o menor numero é {C}")
    if D<C and D<B and D<E:
        print(f"o menor numero é {D}")
    if E<C and E<D and E<E:
        print(f"o menor numero é {E}")
if C>A and C>B and C>D and C>E:
    print(f"o maior numero é {C}")
    if A<C and A<D and A<E:
        print(f"o menor numero é {A}")
    if B<C and B<D and B<E:
        print(f"o menor numero é {B}")
    if D<C and D<B and D<E:
        print(f"o menor numero é {D}")
    if E<C and E<D and E<E:
        print(f"o menor numero é {E}")
if D>A and D>C and D>B and D>E:
    print(f"o maior numero é {D}")
    if A<C and A<D and A<E:
        print(f"o menor numero é {A}")
    if C<B and C<D and C<E:
        print(f"o menor numero é {C}")
    if B<C and B<A and B<E:
        print(f"o menor numero é {B}")
    if E<C and E<D and E<E:
        print(f"o menor numero é {E}")
if E>A and E>C and E>D and E>B:
    print(f"o maior numero é {E}")
    if A<C and A<D and A<E:
        print(f"o menor numero é {A}")
    if C<B and C<D and C<E:
        print(f"o menor numero é {C}")
    if D<C and D<B and D<E:
        print(f"o menor numero é {D}")
    if B<C and B<D and B<E:
        print(f"o menor numero é {B}")